frontend/i18n/en-US.json

```diff
@@ -847,5 +847,13 @@
         "title": "Our Supporters"
       }
     }
+  },
+  "global": {
```

Comment:
Hey @bmartins-42! FYI this will get switched after this gets merged in. I made the decision to flatten the i18n JSON so that we can search for the keys and their usages in a full codebase search. This will go into the top most `_common` name scheme :) More details for this are in #511.

Reply:
Come to think of it though, `_global` makes more sense :) Thanks for the suggestion!